app/models/faq_group.rb

```diff
@@ -0,0 +1,8 @@
+class FaqGroup < ApplicationRecord
+  has_many :faqs, dependent: :destroy
+
+  def number_of_faqs
```

Comment:
No, I opted not to use it in the end. Will remove. Was dumb from the start, a .count works just fine.